src/js/view/controls/next-display-icon.js

```diff
@@ -0,0 +1,30 @@
+define([
+    'utils/helpers',
+    'utils/ui',
+    'templates/display-icon.html'
```

Comment:
I think we loose too much; formatting HTML just with strings is extremely tedious. I suppose the actual problem is avoiding the need for handlebars when running unit tests, which is caused by requiring these templataes